docs/build.md

```diff
@@ -280,19 +280,25 @@ cmake -B build -DGGML_VULKAN=ON
 cmake --build build --config Release
 ```
 
-**With docker**:
+**With containers**:
 
 You don't need to install Vulkan SDK. It will be installed inside the container.
 
-```sh
 # Build the image
-docker build -t llama-cpp-vulkan --target light -f .devops/vulkan.Dockerfile .
+
+<details><summary>Docker example</summary>docker build -t llama-cpp-vulkan --target light -f .devops/vulkan.Dockerfile .</details>
```

Comment:
markdown code block is broken here

also, what I mean is that only the podman command should be put into `details` block, as not many users use it

most users use docker, so let's make it a bit more straight-forward for them.